PowerShellTools.HostService/ServiceManagement/Debugging/PowershellBreakpointRecord.cs

```diff
@@ -0,0 +1,48 @@
+﻿using PowerShellTools.Common.ServiceManagement.DebuggingContract;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace PowerShellTools.HostService.ServiceManagement.Debugging
+{
+    /// <summary>
+    /// Powershell breakpoint wrapper class with breakpoint Id.
+    /// The Id is going to be queried and used for enable/disable/remove breakpoint.
+    /// The reason the Id is not directly defined in [DataContract]PowerShellBreakpoint is due to the Id only belongs to powershell runspace,
+    /// VS does not have this info and does not care either.
+    /// </summary>
+    internal sealed class PowershellBreakpointRecord
+    {
+        private PowershellBreakpoint _psBreakpoint;
+        private int _id;
+
+        public PowershellBreakpointRecord(PowershellBreakpoint bp, int id)
+        {
+            _psBreakpoint = bp;
+            _id = id;
+        }
+
+        /// <summary>
+        /// Breakpoint
+        /// </summary>
+        public PowershellBreakpoint PSBreakpoint

```

Comment:
Not seeing much difference in this case given this is standalone internal class, however I don't mind to change them into internal to keep it consistent with class accessibility.